IPython/utils/tests/test_tokenutil.py

```diff
@@ -13,7 +13,7 @@ def expect_token(expected, cell, cursor_pos):
         if offset + len(line) >= cursor_pos:
             break
         else:
-            offset += len(line)
+            offset += len(line)+1
```

Comment:
Fix error message in case of test faillure, eg if `c==22` it currently show `map(|)` , instead of `|map()`